library/src/G4CMPCrystalGroup.cc

```diff
@@ -141,7 +144,9 @@ G4bool G4CMPCrystalGroup::FillOrthorhombic(G4double Cij[6][6]) const {
 G4bool G4CMPCrystalGroup::FillRhombohedral(G4double Cij[6][6]) const {
   G4double C11=Cij[0][0], C12=Cij[0][1], C13=Cij[0][2], C14=Cij[0][3];
   G4double  C33=Cij[2][2], C44=Cij[3][3], C66=0.5*(C11-C12);
-//Filling the other terms of the matrix, one-by-one (IH)
+//Filling the second order elastic constant for Rhombohedral I Class with Point group 32, 3m and 3m.
```

Comment:
This line is longer than 80 characters.  Please insert a break after the word "Point", so that a new line starts with "// group".  Please insert a space before the uppercase "F" at the beginning of the comment.

Reply:
I change those lines like this :
// Filling the second order elastic constant for Rhombohedral I Class with
// Point group 32, 3m and 3m.
// Filling explicitly the  36 components of the second order elastic tensor.